proposals/0001-proposals.md

```diff
@@ -0,0 +1,37 @@
+---
+author: Jasper Van der Jeugt
+date-accepted: N/A
+discussion: https://github.com/haskell-org/committee/pull/1
+---
+
+# Haskell.org proposals
+
+In an effort to make the work of Haskell.org committee more transparent, we
+would like to adopt a proposal process similar to the [GHC Steering Committee].
+
+[GHC Steering Committee]: https://github.com/ghc-proposals/ghc-proposals
+
+The proposed process is fairly light:
+
+1.  New proposals are created as pull requests with a single file, following
+    a `proposals/XYZW-some-title.extention` naming scheme.  A template is
+    available in `proposals/0000-template.md`.
+
+2.  Proposals must have an author set who is responsible for driving the
```

Comment:
Should we call out here that it is the responsibility of the author to ensure that the various communities (for example: reddit, the discourse, the mailing list) have been notified, if necessary, of the proposal so that the broader community is aware of the proposal and is therefore better able to offer feedback?

Reply:
That seems like something we as a committee should either do ourselves or help with pretty actively. There are so many different communities and sub-communities, and different proposals are going to interest different people, so it makes sense to have some centralized expertise in how to reach out to "the community".

Reply:
Yes, I'll add a paragraph to that.  I don't want to set the communities we need to contact in stone, since different proposals interest different folks (e.g. this one mostly just pertains to the committee).